### PR TITLE
[K8s] Kubernetes yaml files update

### PR DESCRIPTION
**Pull request by kevin-zhonghao**

### CODE REVIEW

File: kubernetes/services/node_manager.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ data:
   application.properties: |
     spring.redis.host=localhost
     spring.redis.port=6380
-    ignite.host=10.99.135.32
+    ignite.host=ignite-ip-service.default.svc.cluster.local
```
Comment (xieus) [inline kubernetes/services/node_manager.yaml:9]: It seems that we are sharing the same Ignite service in this version. Could we rename it to ignite-alcor-service?

File: kubernetes/services/api-gateway.yaml [hunk 2]
```diff
@@ -25,7 +25,7 @@ spec:
   selector:
     matchLabels:
       app: apimanager
-  replicas: 1
+  replicas: 3
   #replicas: 3
```
Comment (xieus) [inline kubernetes/services/api-gateway.yaml:29]: Remove the commented line.

File: kubernetes/services/node_manager.yaml (hunk 1 above)
Comment (kevin-zhonghao) [reply kubernetes/services/node_manager.yaml:9]: Sure

File: kubernetes/services/api-gateway.yaml (hunk 2 above)
Comment (kevin-zhonghao) [reply kubernetes/services/api-gateway.yaml:29]: okay

File: kubernetes/services/port_manager.yaml [hunk 3]
```diff
@@ -4,15 +4,16 @@ metadata:
...
-    microservices.route.service.url=http://10.213.43.166:30003/routes
-    microservices.node.service.url=http://10.213.43.166:30009/nodes
+    microservices.vpc.service.url=http://vpcmanager-service.default.svc.cluster.local:9001
+    microservices.subnet.service.url=http://subnetmanager-service.default.svc.cluster.local:9002
+    microservices.ip.service.url=http://ipmanager-service.default.svc.cluster.local:9004/ips
+    microservices.mac.service.url=http://macmanager-service.default.svc.cluster.local:9005/macs
+    microservices.sg.service.url=http://sgmanager-service.default.svc.cluster.local:9008/securitygroups
+    microservices.route.service.url=http://routemanager-service.default.svc.cluster.local:9003/routes
+    microservices.node.service.url=http://nodemanager-service.default.svc.cluster.local:9007/nodes
+    microservices.dataplane.service.url=http://localhost:8080/v4/port/
```
Comment (xieus) [inline kubernetes/services/port_manager.yaml:16]: We will need to talk to DPM service url. This has dependency on the DPM #240.

**Pull request merged**